test/complexNames.wast.from-wast

```diff
@@ -0,0 +1,10 @@
+(module
+ (type $0 (func))
+ (export "$foo\20\28.bar\29" (func $1))
+ (func $foo\20\28.bar\29 (; 0 ;) (type $0)
+  (nop)
+ )
+ (func $1 (; 1 ;) (type $0)
```

Comment:
interesting that the name ended up `$1` here - I guess that's a limitation of the s-expression parsing?

Reply:
I think it is default name that is coming from https://github.com/WebAssembly/binaryen/blob/master/src/wasm/wasm-binary.cpp#L1034

Reply:
Got it, makes sense.